### PR TITLE
Linking Dashboards docs-220644

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/dashboards-new/link-dashboards.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: Linking Multiple Dashboards (New)
 sidebar_label: Linking Dashboards (New)
 description: Learn how to quickly link dashboards together to view related data.
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Link Multiple Dashboards Together
```
Comment (JainM6) [inline docs/dashboards-new/link-dashboards.md:9]:
```suggestion
## Linking multiple dashboards
```

File: docs/dashboards-new/link-dashboards.md [hunk 2]
```diff
@@ -22,15 +23,14 @@ When you click a data point on a Dashboard (New) chart the Summary tab provides
 * The following image shows a Summary panel from a selected log data point. The **Linked Dashboards** section is at the bottom.<br/>  ![summary v2 logs.png](/img/dashboards-new/drill-root-causes/summary-v2-logs.png)
 * The following image shows a Summary panel from a selected metric data point. The **Linked Dashboards** section is at the bottom.<br/>  ![metrics summary v2.png](/img/dashboards-new/drill-root-causes/metrics-summary-v2.png)
 
-
 ## Link a Dashboard (New) to Explore
```
Comment (JainM6) [inline docs/dashboards-new/link-dashboards.md:26]:
```suggestion
## Linking a dashboard (new) to explore
```

File: docs/dashboards-new/link-dashboards.md [hunk 3]
```diff
@@ -42,19 +42,21 @@ The keys assigned to the Dashboard (New) Stack Linking would need to include tho
...
-1. Enter a **Key** that describes the dashboard.
-1. Next enter **Value(s)**, the component to which the dashboard will be linked in the Explore hierarchy.<br/>![dashboard stack linking august 2021.png](/img/dashboards-new/link-dashboard-explore/dashboard-stack-linking-august-2021.png) 
-1. You can add more key-value pairs by clicking the **Add Link** link.
-1. The checkbox to **Set as default dashboard for this entity in** a hierarchy is available to set when you have defined a specific **Domain**. When set, you can limit the scope to a subset     of hierarchies using the dropdown labelled **No matching hierarchies**.
+1. Select an **Entity Type** for which you would like to display the dashboard.
+1. If you would like to further specify for which entities of a given type the dashboard is displayed you can input the relevant **Entity Name** (as displayed in the Explore tab). If you would like to see the dashboards for all entities of a given type you can use (“*”) as the entity name. <br/><img src={useBaseUrl('img/dashboards-new/link-dashboard-explore/dashboard-stack-linking-may-2023.png')} alt="time-series-or-categorical" width="600"/>
```
Comment (kimsauce) [inline docs/dashboards-new/link-dashboards.md:52]:
```suggestion
1. If you would like to further specify for which entities of a given type the dashboard is displayed, you can input the relevant **Entity Name** (as displayed in the Explore tab). If you would like to see the dashboards for all entities of a given type you can use (`*`) as the entity name. <br/><img src={useBaseUrl('img/dashboards-new/link-dashboard-explore/dashboard-stack-linking-may-2023.png')} alt="time-series-or-categorical" width="600"/>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fixes

File: docs/dashboards-new/link-dashboards.md (hunk 2 above)
Comment (JV0812) [reply docs/dashboards-new/link-dashboards.md:26]: @JainM6 Dashboard (New) and Explore are UI terms, I have updated the heading with more meaningful info.

**Review by kimsauce — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**